doc/nrf/protocols/multiprotocol/index.rst

```diff
@@ -46,3 +46,59 @@ Given the nature of the dynamic multiprotocol solution, take into account the fo
   The application logic must take this into account by providing a callback for all crucial transmissions and handle the ``MAC_TRANSACTION_OVERFLOW (== 241)`` status code.
 * The most time-critical part of the Zigbee stack operation is the Base Device Behavior (BDB) commissioning process.
   Postpone time-consuming operations until the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` or ``ZB_BDB_SIGNAL_DEVICE_REBOOT`` signal is received.
+
+.. _multiprotocol_bt_thread:
+
+Bluetooth LE and Thread coexistence
+***********************************
+
+The Thread protocol has a lower priority than Bluetooth LE.
+When Bluetooth LE consumes a significant amount of radio time, such as during Bluetooth LE central start scanning or to maitain a large number of connections, this can lead to insufficient time left for Thread to access the radio.
```

Comment:
```suggestion
When Bluetooth LE consumes a significant amount of radio time, such as during Bluetooth LE central start scanning or to maintain a large number of connections, this can lead to insufficient time left for Thread to access the radio.
```